### PR TITLE
navbar add

### LINKED ISSUE #11: Navigation
- src/components/Navigation.jsx
- map through the navigation in src/data/navigation.js
image

on hover change the text to blue

image

on select 

image

hover on project there is a drop-down, you can use the [documentation](https://react-bootstrap.netlify.app/docs/components/navbar/) to see how to do it
image

### PR DESCRIPTION
**Pull request by jvarCS**

### CODE REVIEW

File: src/components/Navigation.jsx [hunk 1]
```diff
@@ -1,5 +1,96 @@
...
+                      className={`hover:cursor-pointer rounded-full mb-0 py-1 px-4 no-underline !text-black text-2xl whitespace-nowrap !font-normal hover:!text-blue-600 duration-300 ${
+                        selected === "PROJECTS" &&
+                        "!bg-ewb-blue-200 rounded-full text-white hover:!text-white" // To remove blue hover i set new hover to white
+                      }`}
+                    >
+                      {item.name}
+                    </span>
+                  }
+                >
+                  <NavDropdown.Item
```
Comment (menthy-wu) [inline src/components/Navigation.jsx]: you can map through the sub tabs

File: src/components/Navigation.jsx [hunk 2]
```diff
@@ -1,5 +1,96 @@
...
+                  >
+                    Air Quality System
+                  </NavDropdown.Item>
+                </NavDropdown>
+              ) : (
+                <Nav.Link
+                  as={Link}
+                  key={index}
+                  href={item.link}
+                  onClick={() => setSelected(item.name)} // Blue border when selected became squared immediately after selecting something else so applied rounded-full
```
Comment (menthy-wu) [inline src/components/Navigation.jsx]: remove the comments

File: src/components/Navigation.jsx [hunk 3]
```diff
@@ -1,5 +1,96 @@
...
+                  className={`hover:cursor-pointer rounded-full mb-0 py-1 px-4 no-underline !text-black text-2xl whitespace-nowrap !font-normal hover:!text-blue-600 duration-300 ${
+                    selected === item.name &&
+                    "!bg-ewb-blue-200 rounded-full text-white hover:!text-white" // To remove blue hover i set new hover to white
```
Comment (menthy-wu) [inline src/components/Navigation.jsx]: remove the comments

File: src/components/Navigation.jsx (hunk 1 above)
Comment (jvarCS) [reply src/components/Navigation.jsx]: image

File: src/components/Navigation.jsx (hunk 2 above)
Comment (jvarCS) [reply src/components/Navigation.jsx]: image

File: src/components/Navigation.jsx (hunk 3 above)
Comment (jvarCS) [reply src/components/Navigation.jsx]: image

**Review by menthy-wu — APPROVED** [review]

**Pull request merged**